.changeset/stupid-masks-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@infra/resources": minor
+---
+
+Add user Support PagoPA to ApiTrialSupport.
```

Comment:
**suggestion**
```suggestion
Add user Support PagoPA to `ApiTrialSupport` and create subscription key
```

Reply:
Resolved in eda5c7e8c8214cd1ce743bd5145934c78ab591bf